test/index.js

```diff
@@ -741,6 +741,42 @@ it('accepts request object in validateFunc', (done) => {
     });
 });
 
+it('includes additional attributes in WWW-Authenticate header', (done) => {
+    const server = new Hapi.Server();
```

Comment:
This is already built into [lab](https://github.com/hapijs/lab/tree/master/lib/linters/eslint). `npm test` runs the linter too.